mne/utils.py

```diff
@@ -563,17 +563,39 @@ def estimate_rank(data, tol='auto', return_singular=False, norm=True):
         norms = _compute_row_norms(data)
         data /= norms[:, np.newaxis]
     s = linalg.svd(data, compute_uv=False, overwrite_a=True)
+    rank = _estimate_rank_from_s(s, tol)
+    if return_singular is True:
+        return rank, s
+    else:
+        return rank
+
+
+def _estimate_rank_from_s(s, tol='auto'):
```

Comment:
can we use `_estimate_rank_from_singular_values`? or is `s` a well-known shorthand for `singular_values` in svd decomposition (I've no idea).

Reply:
s is standard